icons/shield40_badge_green_2.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg">
+ <path d="M 9.9997048,0.50008871 C 10.771647,1.1577175 11.772514,1.5546812 12.866059,1.5546812 c 1.093546,0 2.094423,-0.3970141 2.866355,-1.05469344 L 19.499609,4.2672259 C 18.806353,5.2526295 18.3993,6.454039 18.3993,7.7503995 c 0,0.9324568 0.210533,1.8156313 0.586706,2.6047335 0.325025,0.682209 0.513573,1.459591 0.513573,2.265654 0,2.894753 -2.346585,5.241263 -5.241205,5.241263 h -0.822667 c -1.387197,0 -2.625094,0.638651 -3.4359921,1.637965 C 9.1889085,18.500721 7.9509199,17.86205 6.5637234,17.86205 H 5.7410559 c -2.8947204,0 -5.24120446,-2.346611 -5.24120446,-5.241263 0,-0.806023 0.18856666,-1.583455 0.51362206,-2.265654 0.3761441,-0.7891122 0.5867068,-1.6722866 0.5867068,-2.6047335 0,-1.2963605 -0.4070839,-2.4977296 -1.10030867,-3.4831736 L 4.2670662,0.49998776 C 5.0390084,1.1576166 6.0398754,1.5546812 7.1334208,1.5546812 c 1.0935453,0 2.0944224,-0.3969132 2.8663546,-1.05459249 z" fill="#fff" fill-rule="evenodd" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3.864" stroke-width="1.178" style="stroke:#fff;stroke-width:1;stroke-miterlimit:3.864;stroke-dasharray:none;stroke-opacity:1;fill:#006747"/>
```

Comment:
Unlike the U.S. Route shield, Chile’s highway shield is rounded in the center-top. There are only two points at the top, with a curve in the middle.

[<img src="https://upload.wikimedia.org/wikipedia/commons/2/2b/Ruta_5_%28Chile-1R%29.svg" width="64" alt="5">](https://commons.wikimedia.org/wiki/File:Ruta_5_(Chile-1R).svg)

Reply:
A subtle distinction, but here it is:
![Screenshot from 2022-07-06 07-25-05](https://user-images.githubusercontent.com/1732117/177539836-427026fa-bc74-42ae-a88d-96c4f0b96b5e.png) ![Screenshot from 2022-07-06 07-25-18](https://user-images.githubusercontent.com/1732117/177539841-fa37dc7b-1925-47bd-8e3f-e2d3fc39cc04.png)